contrib/curl-cmake/CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
 set (CURL_DIR ${ClickHouse_SOURCE_DIR}/contrib/curl)
+set (CURL_LIBRARY ${ClickHouse_SOURCE_DIR}/contrib/curl/lib)
```

Comment:
This should contain the library name, not the path, i.e. simply `curl`

Right now the [build](https://clickhouse-builds.s3.yandex.net/11300/269fc138be07ac737309f0b403427577dd07ab6a/clickhouse_build_check/build_log_708906849_1593076304.txt) reports the following warnings:

```
2020-06-25 08:39:30 WARNING: Target "sentry" requests linking to directory "/build/contrib/curl/lib".  Targets may link only to libraries.  CMake is dropping the item.
2020-06-25 08:39:30 WARNING: Target "daemon" requests linking to directory "/build/contrib/curl/lib".  Targets may link only to libraries.  CMake is dropping the item.
2020-06-25 08:39:30 WARNING: Target "clickhouse" requests linking to directory "/build/contrib/curl/lib".  Targets may link only to libraries.  CMake is dropping the item.
2020-06-25 08:39:30 WARNING: Target "clickhouse-server-lib" requests linking to directory "/build/contrib/curl/lib".  Targets may link only to libraries.  CMake is dropping the item.
2020-06-25 08:39:30 WARNING: Target "clickhouse-local-lib" requests linking to directory "/build/contrib/curl/lib".  Targets may link only to libraries.  CMake is dropping the item.
2020-06-25 08:39:30 WARNING: Target "clickhouse-copier-lib" requests linking to directory "/build/contrib/curl/lib".  Targets may link only to libraries.  CMake is dropping the item.
2020-06-25 08:39:30 WARNING: Target "clickhouse-odbc-bridge" requests linking to directory "/build/contrib/curl/lib".  Targets may link only to libraries.  CMake is dropping the item.
2020-06-25 08:39:30 WARNING: Target "clickhouse-odbc-bridge-lib" requests linking to directory "/build/contrib/curl/lib".  Targets may link only to libraries.  CMake is dropping the item.
2
```

P.S. everything works because of the following explicit linking with curl:

```cmake
if (USE_SENTRY)
    target_link_libraries (daemon PRIVATE curl) # <<
    target_link_libraries (daemon PRIVATE ${SENTRY_LIBRARY})
endif ()
```

Reply:
@azat I have the same issue. Could you please make a pull request with fix?

Reply:
#12041